src/lib/block/kuznyechik/kuznyechik.cpp

```diff
@@ -47,4103 +48,91 @@ alignas(256) const uint8_t IS[256] = {
-                                  {0x8fa1f32cea5616de, 0x8ab8739c6a431d59}, {0x46854fb74022bac2, 0x948abc2eb3d6e797},
-                                  {0xaad2ef571e63ca4e, 0x02237ec88f70af58}, {0xf24194a35cf5b03f, 0xda891e6ad65de88d},
-                                  {0x85125eac553a560e, 0x0f1986598dab7d67}, {0x04212d3f9838fe1f, 0x3d9462c3d7d18173},
-                                  {0x03686d816a12a199, 0x616fc8c0cf2c10b4}, {0xb21401d5c8f38b0c, 0x8cddf10738d32fb1},
-                                  {0xa290b529ed13f570, 0x78c8ba8de2116ebe}, {0x41cc0f09b208e544, 0xc871162dab2b7650},
-                                  {0xbe777694a3bb4a2d, 0xcba2578182636f24}, {0x2be19cc4d361625f, 0x30ae9a52d50a534c},
-                                  {0x7e880a0edcb10778, 0x315ea5367332e560}, {0x609e3e4dde05c7cb, 0x7d7e79ba99c94522},
-                                  {0x9ebcddafe9b8b6d5, 0xe088c156f524ed3a}, {0x6f95248ddf5fa773, 0x5b6e17fcec551503},
-                                  {0xe6e40d60e12d305c, 0x13083723db4e28f1}, {0x79c14ab02e9b58fe, 0x6da50f356bcf74a7},
-                                  {0xa6b19816752b0b6f, 0x455cd84e35c0efcd}, {0x902e5d10ceec081a, 0x58bd5650c51d0c77},
-                                  {0xa3092f56cb1d2b07, 0xe6ed43cda7b4dfd2}, {0x02f1f7fe4c1c7fee, 0xff4a31808a89a1d8},
-                                  {0x6cfd490cb54d06ea, 0x3a01df3c237905b7}, {0xad9bafe9ec4995c8, 0x5ed8d4cb978d3e9f},
-                                  {0xec57a0e05e41708c, 0x96a9c2e63ca648cf}, {0x471cd5c8662c64b5, 0x0aaf456ef67356fb},
-                                  {0x5148bbf597e89b38, 0x3c645da771e9375f}, {0x72eb7d4fb7f9c659, 0x762103b0c982a5f5},
-                                  {0xffbb799d11b3af69, 0x03d341ac29481974}, {0x7b30bd4e62872710, 0x92ef3eb5e146d57f},
-                                  {0xfcd3141c7ba10ef0, 0x62bc896ce66409c0}, {0x6526890d6033e7a3, 0xdecfe2390bbd753d},
-                                  {0x261b71fa9e277d09, 0xe9f4c5942a1fa2b5}, {0x4817cf086776040d, 0x2cbf2b2883ef06da},
-                                  {0xb4c4db141cd70afd, 0x4e03a244658b0f1a}, {0x6a2d93cd6169871b, 0xf8df8c7f7e21251c},
-                                  {0x556996ca0fd06527, 0x01f03f64a638b62c}, {0x16546e3df1c4ff8d, 0x36cb18c9879a61a4},
-                                  {0xcbd54b65e668d3f2, 0xe178fe32531c5b16}, {0x1275430269fc0192, 0x0b5f7a0a504be0d7},
-                                  {0x91b7c76fe8e2d66d, 0xc698af1080b8bd1b}, {0x1ce7c3bd4ea8bf5d, 0xb36aed0c6072019a},
-                                  {0x06d0dac1d42481f1, 0xc2de53435d5820ab}, {0x0c6377416b48c121, 0x477fa686bab04095},
-                                  {0x2e592b846d574237, 0x931f01d1477e6353}, {0x2fc0b1fb4b599c40, 0x0d3af89102dbd23f},
-                                  {0xdb51ff99c388ad8e, 0x156db5b889de1a19}, {0x5bfb16752884dbe8, 0xb9c5a86296015761},
-                                  {0xe9ef17a0e07750e4, 0x35185965aed278d0},
```

Comment:
```suggestion
constexpr auto T = T_table();
constexpr auto IT = IT_table();
```

?

Reply:
That was the plan but it hits constexpr expression limits with Clang.

However I just pushed a new versoin that at least for my Clang does allow compile time initialization. Let's see what CI thinks..

Reply:
It worked!